yarn.lock

```diff
@@ -9387,14 +9387,6 @@ multibase@~0.6.0:
     base-x "^3.0.8"
     buffer "^5.5.0"
 
-multicodec@2.1.0:
```

Comment:
maybe regenerate the whole `yarn.lock` instead of deleting this area - lock files are not meant to be edited

Reply:
This is what I've done ;-)